### PR TITLE
Updates files to set max consoles limit from 64MB to 16MB

### PR DESCRIPTION
**Pull request by himanshu0503**
https://github.com/Shippable/cexec/issues/143

### CODE REVIEW

**Comment by himanshu0503** [comment]
https://rcapp.shippable.com/github/himanshu0503/5134/runs/110/1/console

File: script_runner.py [hunk 1]
```diff
@@ -19,7 +19,11 @@ def __init__(self, job_id, shippable_adapter, \
         self.console_buffer = []
         self.console_buffer_lock = threading.Lock()
         self.continue_trigger_flush_console_output = True
-        self.max_consoles_size_in_bytes = 64 * 1024 * 1024  # 64 MB
+        self.max_consoles_size_MB = self.config['MAX_CONSOLES_SIZE_MB']
+        if self.max_consoles_size_MB:
```
Comment (ric03uec) [inline script_runner.py]: all this logic should happen in config.py
this file should just use the value in self.max_console_size

File: script_runner.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def __init__(self, job_id, shippable_adapter, \
         self.console_buffer = []
         self.console_buffer_lock = threading.Lock()
         self.continue_trigger_flush_console_output = True
-        self.max_consoles_size_in_bytes = 64 * 1024 * 1024  # 64 MB
+        self.max_consoles_size_in_bytes = self.config['MAX_CONSOLES_SIZE_MB'] * 1024 * 1024
```
Comment (ric03uec) [inline script_runner.py]: why not convert this to bytes upfront in config.py. we shouldn't be tweaking this value once its processed

**Comment by himanshu0503** [comment]
https://rcapp.shippable.com/github/himanshu0503/5134/runs/125/1/console

**Pull request merged**